google/cloud/pubsublite/internal/stream_factory.h

```diff
@@ -0,0 +1,85 @@
+#include "absl/container/flat_hash_map.h"
```

Comment:
nit: missing copyright boiler plate.  This is required for OSS stuff.

Reply:
Done.